src/style/content/conversation/message-list.less

```diff
@@ -274,7 +274,7 @@
 // MESSAGE - BODY
 .message-body {
   position: relative;
-  max-width: calc(100% - var(--conversation-message-sender-width));
+  max-width: 100%;
   padding-left: var(--conversation-message-sender-width);
```

Comment:
` var(--conversation-message-sender-width)` (the left gap for a user's avatar) is already taken care of with padding here